test/validate_v1_v2_equality.ts

```diff
@@ -1,36 +1,244 @@
+  });
+
+  it('deck_limit matches', () => {
+    validatev1V2Equality('deck_limit',
+        function(title) { return v1CardsByTitle.get(title).deck_limit; }, 
+        function(title) { return v2CardsByTitle.get(title).deck_limit; }); 
+  });
+
+  it('faction_id matches', () => {
+    validatev1V2Equality('faction_id',
+        function(title) { return v1CardsByTitle.get(title).faction_code; }, 
+        function(title) { return v2CardsByTitle.get(title).faction_id; }); 
+  });
+
+  it('influence_cost matches', () => {
+    validatev1V2Equality('influence_cost',
+        function(title) { return v1CardsByTitle.get(title).faction_cost; }, 
+        function(title) { return v2CardsByTitle.get(title).influence_cost; }); 
+  });
+
+  it('influence_limit matches', () => {
+    validatev1V2Equality('influence_limit',
+        function(title) { return v1CardsByTitle.get(title).influence_limit; }, 
+        function(title) { return v2CardsByTitle.get(title).influence_limit; }); 
+  });
+
+  it('memory_cost matches', () => {
+    validatev1V2Equality('memory_cost',
+        function(title) { return v1CardsByTitle.get(title).memory_cost; }, 
+        function(title) { return v2CardsByTitle.get(title).memory_cost; }); 
+  });
+
+  it('minimum_deck_size matches', () => {
+    validatev1V2Equality('minimum_deck_size',
+        function(title) { return v1CardsByTitle.get(title).minimum_deck_size; }, 
+        function(title) { return v2CardsByTitle.get(title).minimum_deck_size; }); 
+  });
+
+  it('side_id matches', () => {
+    validatev1V2Equality('side_id',
+        function(title) { return v1CardsByTitle.get(title).side_code; }, 
+        function(title) { return v2CardsByTitle.get(title).side_id; }); 
+  });
+
+  it('strength matches', () => {
+    validatev1V2Equality('strength',
+        function(title) { return v1CardsByTitle.get(title).strength; }, 
+        function(title) { return v2CardsByTitle.get(title).strength; }); 
+  });
+
+  it('trash_cost matches', () => {
+    validatev1V2Equality('trash_cost',
+        function(title) { return v1CardsByTitle.get(title).trash_cost; }, 
+        function(title) { return v2CardsByTitle.get(title).trash_cost; }); 
+  });
+
+  it('is_unique matches', () => {
+    validatev1V2Equality('trash_cost',
+        function(title) { return v1CardsByTitle.get(title).uniqueness; }, 
+        function(title) { return v2CardsByTitle.get(title).is_unique; }); 
+  });
+});
+
+describe('Printings v1/v2 equality', () => {
+  const v1Cards = getCardsJson();
+  const v1CardsByCode = new Map<string, any>();
+  v1Cards.forEach(c => {
+    v1CardsByCode.set(c.code, c);
+  });
+  const printings = getPrintingsV2Json();
+  const printingsById = new Map<string, any>();
+  printings.forEach(p => {
+    printingsById.set(p.id, p);
+  });
+
+  it('correct number of printings', () => {
+    v1CardsByCode.forEach((v, k) => {
+      expect(printingsById.has(k), `v1 code ${k} exists in printings id map`).to.be.true;
```

Comment:
Could also be `expect(print).to.have.keys(k)`

Reply:
there is something funny happening with the typescript map interaction.  That doesn't work here.  Same with using a Set().

Reply:
yeah, that doesn't work either.  it actually makes the test timeout.  🤷